server/run.sh

```diff
@@ -13,8 +13,14 @@ do
     "--test" )
       python -m pytest;;
     "--dev" )
+      redis-server
       export FLASK_APP=halomod_app
       export FLASK_ENV=development
       flask run;;
+    "--prod" )
```

Comment:
is this called somewhere? 
We dont need this for ourselves

Reply:
Removed.